src/extras/primitives/primitives/a-skybox.js

```diff
@@ -0,0 +1,24 @@
+var getMeshMixin = require('../getMeshMixin');
+var registerPrimitive = require('../registerPrimitive');
+var utils = require('../../../utils/');
+
+registerPrimitive('a-skybox', utils.extendDeep({}, getMeshMixin(), {
```

Comment:
feel free to file a follow-up issue for it instead, but it'd be awesome to at least stub out the reference Docs page for this primitive at `docs/primitives/a-skybox.md` (see [`docs/primitives/a-sky.md`](https://github.com/aframevr/aframe/blob/master/docs/primitives/a-sky.md) for an example).